spec/03-plugins/method-restriction/02-access_spec.lua

```diff
@@ -0,0 +1,116 @@
+local helpers = require "spec.helpers"
+local cjson = require "cjson"
+
+describe("Plugin: method-restriction (access)", function()
+  local client
+  setup(function()
+    helpers.kill_all()
```

Comment:
Oh yes, right! Sorry, on mobile now!

> On Jul 20, 2016, at 6:25 PM, Fábio da Silva Ribeiro <notifications@github.com> wrote:
> 
> In spec/03-plugins/method-restriction/02-access_spec.lua:
> 
> > @@ -0,0 +1,116 @@
> > +local helpers = require "spec.helpers"
> > +local cjson = require "cjson"
> > +
> > +describe("Plugin: method-restriction (access)", function()
> > +  local client
> > +  setup(function()
> > +    helpers.kill_all()
> Did you mean the kill_all helper?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>